recipes/zimg/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "zimg" %}
+{% set version = "3.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sekrit-twc/zimg/archive/release-{{ version }}.tar.gz
+  sha256: 219d1bc6b7fde1355d72c9b406ebd730a4aed9c21da779660f0a4c851243e32f
+
+build:
+  skip: True  # [win]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('zimg', min_pin='x.x', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - automake
+    - libtool
+    - make
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libzimg${SHLIB_EXT}  # [not win]
+    - if not exist %PREFIX%\bin\zimg.dll exit 1  # [win]
+
+    # CFEP 18
+    - test ! -f ${PREFIX}/lib/libzimg.a  # [not win]
+    - if exist %PREFIX%\lib\zimg.lib exit 1  # [win]
```

```suggestion
    - if exist %PREFIX%\Library\lib\zimg.lib exit 1  # [win]
```

You might want to change 
```
--prefix=%PREFIX% 
```
to
```
--prefix=%LIBRARY_PREFIX% 
```
in your build script to get this to pass the tests.

---

Thanks for pointing that out. I didn't notice the difference between `%LIBRARY_PREFIX%` and `%PREFIX`.

---

its a common annoyance of conda. I think the original designers agreed that this discrepancy was a mistake.